iceberg/datasets/gdelt/fetch.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
```

Comment:
Should we add validation for expected no of arguments with value format? Like in this case 
`Expecting one argument with epoch or date`